Documentation/Blazorise.Docs/Pages/News/2024-07-15-release-notes-160.razor

```diff
@@ -71,6 +71,17 @@
     Visit our docs for more information, <Anchor To="docs/extensions/chart-zoom">Chart Zoom Plugin</Anchor>
 </Paragraph>
 
+<Heading Size="HeadingSize.Is3">
+    Video update
+</Heading>
+
+<Paragraph>
+    We have updatet our Video extension component to work with the new <Anchor To="https://www.vidstack.io/">VidStack</Anchor> library. Until now we have used a Plyr library, but it was slowly been replaced by a neweset and more modern VidStack library. So,  the obvious reason was for us also to make an update as it provides more features, and is much more stable. Not to mention that it also has many more features.
```

Comment:
- updatet => updated.
- we have used **the** Plyr
- it was slowly been replaced => it has slowly been replaced 
- neweset?
- So,  the obvious reason was for us also to make an update => Since the newer library is much more stable and provides more feature, we've went ahead and updated, implementing the new features, and making the transition as seamless as possible.
- Not to mention that it also has many more features. => Again?

I guess you'll review the notes when you release anyway.